test/mbta_v3_api/alert_test.exs

```diff
@@ -121,6 +123,7 @@ defmodule MBTAV3API.AlertTest do
                ],
                "cause" => "ALIENS",
                "description" => "Description",
+               "duration_certainty" => "UNKNOWABLE",
```

Comment:
Ridiculously minor nitpick:

```suggestion
               "duration_certainty" => "BEYOND_MORTAL_COMPREHENSION",
```

`UNKNOWABLE` is close enough to `UNKNOWN` that I had to read this again to see what the test was; something more obviously distinct might be easier to scan through at a glance.